src/fragments/lib/auth/js/emailpassword.mdx

```diff
@@ -129,7 +243,29 @@ Auth.signUp({
 
 When signing in with user name and password, you will pass in the username and the password to the `signIn` method of the Auth class.
 
-```javascript
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+interface SignInParams {
+  username: string;
+  password: string;
+}
+
+export async function signIn(signInParams: SignInParams) {
```

Comment:
Can you update this to include passing in `username` and `password` as a destructured object such as below:

`signIn({username, password}:signInParams)`